index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <main class="container">        
+        <header>
+            <div class="toolbar">
+                <h2>My Logo</h2>            
+                <img src="images/union.svg" alt="menu" />                 
+            </div>
+        </header>
+
+        <section class="headline">
+            <h2><br>I'm Carlos<br>Glad to see<br>you!</h2>            
+            <p>I’m a software developer! I can help you<br>build a product , feature or website Look<br>through some of my work and<br>experience! If you like what you see and<br>have a project you need coded, don’t<br>hestiate to contact me.</p>
+            <h3>LET'S CONNECT</h3>
+              <div class="icons-social">
+                <img src="images/twitter.png" class="social" alt="twitter" />            
+                <img src="images/Linkedin.png" class="social" alt="Linkedin" />
+                <img src="images/medium.png" class="social" alt="Medium" />
+                <img src="images/github.png" class="social" alt="Github" />            
+                <img src="images/angellist.png" class="social" alt="angellist" />
+              </div>
```

Comment:
- Your list of icons should use a ul tag to group them, and li tag per icon. Each icon should have a link (a tag)